src/components/subHeader/helpers/fetchRestaurantData.js

```diff
@@ -0,0 +1,12 @@
+// api
+import fetchData from "../../../api/fetch";
+
+// constants
+import baseUrl from "../../../constants/baseUrlForData";
+import { RESTAURANT_DATA_END_POINT } from "../../../api/endPoints";
+
+function fetchRestaurantData(restaurantId){
```

Comment:
this looks like redundant function, directly call fetchData(), also you don't need wrapper fetchData, directly call the API and write success handler here itself